src/components/reusables/Form.tsx

```diff
@@ -23,8 +27,12 @@ export default class Form extends React.Component<FormProps, void> {
   };
 
   render(): JSX.Element {
+    let className = this.props.className ? `clearfix ${this.props.className}` : "clearfix";
```

Comment:
If `clearfix` will always be rendered, you could have it in the `className` attribute below.